avm/ptn/sa/modernize-your-code/tests/e2e/enterprise-grade/main.test.bicep

```
@@ -0,0 +1,59 @@
+targetScope = 'subscription'
+
+metadata name = 'Default configuration with WAF aligned parameter values'
+metadata description = 'This instance deploys the [Modernize Your Code Solution Accelerator](https://github.com/microsoft/Modernize-Your-Code-Solution-Accelerator) using parameters that deploy the WAF aligned configuration.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-waf-${namePrefix}-sa.moderncode-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+#disable-next-line no-unused-params // overridden below to avoid the allowed location list validation
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'samycwaf'
```

Since `waf` is used for waf aligned tests, please change the suffix, e.g.,

```suggestion
param serviceShort string = 'samyceg'
```